hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3secret/Application.java

```diff
@@ -15,7 +15,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.apache.hadoop.ozone.s3secret;
+
+import org.glassfish.jersey.server.ResourceConfig;
 
-window.onload = function () {
-    document.getElementById('s3gurl').innerHTML = window.location.origin;
-};
```

Comment:
I guess git finds the license header similar enough to consider this to be a rename and minor change.

`hadoop-ozone/s3gateway/src/main/resources/webapps/static/s3g.js` was removed.  It used to add the show the endpoint URL for S3 access.  Since it used its own address for that, we cannot rely on this due to the port difference.

`hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3secret/Application.java` is a new file, it's a `ResourceConfig` for the `/secret` REST endpoint.